src/SolutionTemplate/UnoSolutionTemplate.WinUI.net6/Skia.WPF/SkiaWpf.winui.net6.vstemplate

```diff
@@ -1,22 +1,40 @@
 <VSTemplate Version="3.0.0" xmlns="http://schemas.microsoft.com/developer/vstemplate/2005" Type="Project">
   <TemplateData>
-    <Name>UnoQuickStart.Skia.Wpf</Name>
-    <Description>No Description</Description>
-    <Icon>__TemplateIcon.png</Icon>
-    <PreviewImage>__PreviewImage.png</PreviewImage>
-    <ProjectType>CSharp</ProjectType>
-    <ProjectSubType />
-    <SortOrder>1000</SortOrder>
-    <DefaultName>UnoQuickStart.Skia.Wpf</DefaultName>
-    <ProvideDefaultName>true</ProvideDefaultName>
-    <EnableLocationBrowseButton>true</EnableLocationBrowseButton>
-    <LocationField>Enabled</LocationField>
-    <CreateInPlace>true</CreateInPlace>
+	<Name>UnoQuickStart.Skia.Wpf.Host</Name>
```

Comment:
suffixing of `.Host` on Name and DefaultName is intended?

Reply:
i guess this is to avoid having to change all the namespaces?

Reply:
It's actually a name that's not used, but I'll change it for clarity, thanks.